### DISCUSSION TITLE
Schedules Direct Problem: No Guide Data

### DISCUSSION THREAD

**Discussion by willieb9000** [OpenSageTV/Community#62] [Guide/EPG]
Hi everyone. I seem to have lost all my guide data.  I have error messages in the logs that say 
"javax.net.ssl.SSLException: java.lang.RuntimeException: Unexpected error: java.security.InvalidAlgorithmParameterException: the trustAnchors parameter must be non-empty"

When I go to select my guide lineup in the Source setup, if I try to select Schedules Direct, I get "There was a problem communicating with the Schedules Direct server."

I see on the new forum (probably located very near this thread) that some changes to SD were contemplated recently, but I can't tell if they were implemented and that is causing the problems. Anyone else having this issue? Any thoughts on how to correct?

**Comment by Sagetvmaster** [comment]
Windows 10 installation of Sage.  All three Schedules Direct listings are AOK here.

**Comment by TGsell1** [comment]
Also Win10 and both OTA and Cablcard listing good.  Sent from my iPhoneOn Aug 14, 2023, at 10:04 AM, Sagetvmaster ***@***.***> wrote:﻿
Windows 10 installation of Sage.  All three Schedules Direct listings are AOK here.

**Comment by Ugadata** [comment]
For the time being, nothing has changed for SchedulesDirect. 
The thread about changes to SchedulesDirect have not happened yet and there is no ETA for when this will be done.

It maybe an Internet issue?   When is your SD subscription due?

**Comment by eelpout** [comment]
just checked and mine is still good out to two weeks (SF Bay Area). 

Maybe it's a regional issue? Have you checked with Schedules Direct? they are good about getting back to you when you have a question.

**Comment by willieb9000** [comment]
Thanks so much for all the replies everyone. Nice to know this little user group is not completely gone yet. I'll keep troubleshooting on my end but it's good to know it is working for everyone else.  I'm running sage inside a docker. I didn't change anything recently so it is a mystery to me.

**Comment by willieb9000** [comment]
Update -  I just installed a Sage Windows Server and was able to pull guide data no problem with my existing SD login. SO definitely an issue with my docker sage installation. Sometimes the dockers will update themselves when a developer pushes out a new version, so I suppose it is possible someone update my Sage docker and there is some sort of issue with SD in it. Will keep working at it.
